src/QueryBuilder/ResultViewer/DynamicTable/DynamicTable.test.js

```diff
@@ -114,5 +146,12 @@ describe('DynamicTable component', () => {
     expect(trueCell).toBeInTheDocument();
     expect(falseCell).toBeInTheDocument();
     expect(trueCell.compareDocumentPosition(falseCell)).toBe(Node.DOCUMENT_POSITION_FOLLOWING);
+
+    const coolDateCell = getByText('1/1/2021'); // after midnight in EST
+    const lessCoolDateCell = getByText('12/31/2020'); // before midnight in EST
+
+    expect(coolDateCell).toBeInTheDocument();
+    expect(lessCoolDateCell).toBeInTheDocument();
+    expect(coolDateCell.compareDocumentPosition(lessCoolDateCell)).toBe(Node.DOCUMENT_POSITION_FOLLOWING);
```

Comment:
Will this break when daylight savings time hits? Since NY will only be 4 hours behind then

Reply:
it should be based on the date being localized itself, not the current date (if it was july 1, it'd be EDT)